Docs: copyedits trial account statements (#28072)
MaterializeInc/materialize: doc/user/content/get-started/quickstart.md

* **Consistency**: results are always correct; never even transiently wrong.

## Prerequisite

A Materialize account to sign into the [Materialize console](https://console.materialize.com/). If you do not have an account, you can [sign up for a free trial account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).

> ## Step 0. Sign into the Console

Reviewer: Isn't this a real step, anyways? You either need to check whether you're signed in, or sign in.

> ## Step 0. Sign into the Console

Author: Ah, yes ... but heh heh ... I made it a Step 0 so that the remaining steps will align with the quickstart instructions inside the Console.  

I'm also going back and forth whether to include it as part of the prereq, but figured when I do a pass on the quickstart content, I'll give it a better think.

> ## Step 0. Sign into the Console

Reviewer: Gotcha.

> Open the [Materialize console](https://console.materialize.com/) and, if not already signed in, sign in. By default, the SQL Shell should display in the console.  You can also select SQL Shell in the console's left-hand menu.

Reviewer: ```suggestion
Navigate to the [Materialize console](https://console.materialize.com/) and sign
in. By default, you should land in the SQL Shell. If you're already signed in,
you can access the SQL Shell in the left-hand menu.
```

Reviewer: If a user signs in for the first time, they'll **always** land in the shell (which might change in a few quarters), so it feels more relevant to just point already signed-in users to the SQL Shell if they're somewhere else!

* **Consistency**: results are always correct; never even transiently wrong.

## Prerequisite

A Materialize account. If you do not have an account, you can [sign up for a free trial](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).

> ## Step 0. Sign into the Console

Reviewer: ```suggestion
## Step 0. Sign in to Materialize
```

DROP TABLE fraud_accounts;
```

> 

Author: Yup -- I only deleted this since at this point in time since there wasn't any content once I removed the convert playground to trial account CTA.

I can definitely add the two CTAs to keep the section.

> 

Reviewer: For context, we previously did not allow playground accounts to create connections and sources; these accounts were limited to load generators. We can now point users to getting started with their own data, since the restriction no longer applies for trial accounts.

> 

Author: Ah ... got it. For now, just added 2 bullet points to click the various buttons in the Console (since one button goes to the Sources in the console - and the other does the chilipiper for the guided tour).  But, definitely will need to be revisited when we do the refresh of the quickstart.

[//]: # "TODO(morsapaes) Extend to suggest third party tools. dbt, Census and Metabase could all fit here to do interesting things as a follow-up."

> - To get started with your own data, click the `Connect data` button in the
>   Console.
> 
> - To get a guided tour, click the `Talk to us` button in the Console.

Reviewer: ```suggestion
To get started ingesting your own data from an external system like Kafka, MySQL
or PostgreSQL, check the documentation for [sources](/sql/create-source/), and
navigate to **Data** > **Sources** > **New source** in the [Materialize Console](https://console.materialize.com/)
to create your first source.
```

Reviewer: This original CTAs make sense for users that are following the embedded quickstart in the console, but not for users who are following the quickstart in the documentation (which _might_ include users running the Docker image, which has no notion of a UI). Left a suggestion to point users to the source creation UI in the console, but also to the `CREATE SOURCE` landing page (which should really be an overview page for `Ingest data`, but we don't have one just yet).
